modules/Hoot/config/domMetadata.js

```diff
@@ -70,8 +72,22 @@ export function layerConflateForm( data ) {
 
                 }
 
-                if (!_isEqual(advOpts, advancedOpts.advancedOptions)) {
-                    advancedOpts.createGroups(advOpts);
+                if (!_isEqual(advOpts, advancedOpts.advancedOptions) || favoriteOptions.length ) {
+
+                    let checkFavs = _filter( favoriteOptions, function(a) {
+                        if ( a.name === type ) {
+                            return a;
+                        }
+                    });
+
+                    if ( type === checkFavs[0].name ) {
```

Comment:
need to account for case where checkFavs is empty. case when a default type is selected an error is console logged